features/step_definitions/basic_steps.rb

```diff
@@ -1,3 +1,20 @@
 Then("I should see {string}") do |content|
   expect(page).to have_content content
 end
+
+Then("{string} is present") do |selector|
+  expect(page).to have_selector :css, "#navigationBar"
+  expect(page).to have_selector :css, "#footer"
+end
+
+Then("I would like to see a {string} link") do |link|
+  find_link link
```

Comment:
Hmm... So you `find` the filnk. And then what? 
(if this is supposed to be an assertion, why not use the `expect` keyword?)

Reply:
The link will be functional later on in the project, is it wrong to just test the presence of it?

Reply:
This is not a test. This is a command to look for a link and nothing else